apps/src/code-studio/components/ShareDialog.jsx

```diff
@@ -53,9 +66,26 @@ var ShareDialog = React.createClass({
       showAdvancedOptions: false,
       exporting: false,
       exportError: null,
+      isTwitterAvailable: false,
+      isFacebookAvailable: false,
     };
   },
 
+  componentDidMount() {
+    if (this.props.canShareSocial) {
```

Comment:
What's the difference between props.canShareSocial and checking for social availability? Is there a name that differentiates them better.